WooCommerce/Classes/ViewRelated/Themes/ThemesCarouselView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+import Kingfisher
+
+
+struct ThemesCarouselView: View {
+    /// Tuple array containing theme name and theme Url string pair.
+    let themesInfo: [(name: String, url: String)] = [
+        (name: "Google", url: "google.com"),
+        (name: "Facebook", url: "facebook.com"),
+        (name: "WordPress", url: "wordpress.com"),
+        (name: "Microsoft", url: "microsoft.com"),
+        (name: "Amazon", url: "amazon.com"),
+        (name: "Apple", url: "apple.com")
+    ]
+
+    /// The title of the message at the end of the carousel.
+    let lastMessageHeading: String
+
+    /// The content of the message at the end of the carousel.
+    let lastMessageContent: String
+
+
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            HStack() {
+
+                // Demo for correct state
+                ForEach(themesInfo, id: \.name) { theme in
+                    if let themeUrl = getThemeUrl(themeUrl: theme.url) {
+                        themeImage(url: themeUrl)
+                    } else {
+                        themeNameCard(name: theme.name)
+                    }
+                }
+
+                // Demo for error state
+                themeNameCard(name: themesInfo[0].name)
+
+                // Message at the end of the carousel
+                lastMessageCard(heading: lastMessageHeading, message: lastMessageContent)
+            }
+        }
+    }
+}
+
+private extension ThemesCarouselView {
+    func themeImage(url: URL) -> some View {
```

Comment:
Nit: how about `themeImageCard` for consistency?

Reply:
Yup! Added in https://github.com/woocommerce/woocommerce-ios/pull/11336/commits/fb5975eda0c7bf0a0c2a54ff170667ff3c3482be